packages/devextreme/testing/tests/DevExpress.ui.widgets/scrollView.tests.js

```diff
@@ -290,6 +291,51 @@ QUnit.module('onReachBottom', () => {
     });
 });
 
+if(QUnit.urlParams['nojquery']) {
+    QUnit.module('ShadowDom', moduleConfig, () => {
+        QUnit.skipInShadowDomMode('dxscrollinit handler should be fired after triggering of wheel event', function(assert) {
+            assert.expect(1);
+
+            const shadowHost = document.querySelector('#qunit-fixture');
+            const shadowRoot = shadowHost.attachShadow({ mode: 'open' });
+            const scrollView = document.createElement('div');
+            const scrollViewContent = document.createElement('div');
+
+            scrollView.setAttribute('style', 'height: 200px; width: 50px; position: absolute; top: 0; bottom: 300px; padding: 20px;');
+            scrollViewContent.setAttribute('style', 'height: 400px; width: 50px;');
+
+            scrollView.appendChild(scrollViewContent);
+            shadowRoot.appendChild(scrollView);
+
+            const $scrollView = $(scrollView).dxScrollView({
+                useNative: false,
+            });
+
+            const wrapper = $scrollView.find(`.${SCROLLABLE_WRAPPER_CLASS}`).get(0);
+
+            eventsEngine.on(wrapper, SCROLL_INIT_EVENT, () => {
+                assert.ok(true, 'dxscrollinit was fired');
+            });
+
+            const originalEvent = {
+                target: shadowHost,
+                composedPath: () => [
+                    scrollViewContent,
+                    scrollView,
+                    shadowRoot,
+                    shadowHost,
+                ],
+            };
+
+            pointerMock(scrollViewContent)
+                .start()
+                ._wheel({
+                    target: shadowHost,
+                    originalEvent: $.Event('wheel', originalEvent),
+                });
```

Comment:
Why cannot we write more functional test with real scrolling? Like just usual trigging of a wheel events, i'm pretty sire we have similar tests for scrollview/scrollable